tests/cli/test_export_model.py

```diff
@@ -43,14 +44,17 @@ def test_export(monkeypatch, tmp_path, path):
 
 
 @pytest.mark.parametrize("output", [None, "exported.pt"])
-def test_export_cli(monkeypatch, tmp_path, output):
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64])
+def test_export_cli(monkeypatch, tmp_path, output, dtype):
     """Test that the export cli runs without an error raise."""
     monkeypatch.chdir(tmp_path)
+
+    dtype_string = "64" if dtype == torch.float64 else "32"
```

Comment:
Maybe a bit more general if we also want to test 16 bit a some point.

```suggestion
    dtype_string = str(dtype)[-2:]
```